hedera-node/src/test/java/com/hedera/services/tokens/HederaTokenStoreTest.java

```diff
@@ -1584,6 +1595,7 @@ public void commitPutsToMapAndClears() {
 		// and:
 		assertTrue(subject.symbolKeyedIds.containsKey(symbol));
 		assertTrue(subject.nameKeyedIds.containsKey(name));
+		assertTrue(subject.isKnownTreasury(treasury));
 		assertEquals(created, subject.symbolKeyedIds.get(symbol));
 		assertEquals(created, subject.nameKeyedIds.get(name));
```

Comment:
you can add an additional check that checks that the created object is populated in the `knownTreasuries` map similar to the symbol and name